storage/src/vespa/storage/storageserver/communicationmanager.cpp

```diff
@@ -49,13 +48,14 @@ CommunicationManager::receiveStorageReply(const std::shared_ptr<api::StorageRepl
 }
 
 namespace {
-    vespalib::string getNodeId(StorageComponent& sc) {
-        vespalib::asciistream ost;
-        ost << sc.cluster_context().cluster_name() << "/" << sc.getNodeType() << "/" << sc.getIndex();
-        return ost.str();
-    }
 
-    framework::SecondTime TEN_MINUTES(600);
+vespalib::string getNodeId(StorageComponent& sc) {
+    vespalib::asciistream ost;
+    ost << sc.cluster_context().cluster_name() << "/" << sc.getNodeType() << "/" << sc.getIndex();
+    return ost.str();
+}
+
+vespalib::duration TEN_MINUTES = 600s;
```

Comment:
Also, since this is related to "timing out" stale protocol generations, maybe we should rename it to something like `STALE_PROTOCOL_LIFETIME`. While we're at it, bumping from 60s to 60 minutes also seems reasonable since not a lot of memory should be held up here, and the consequences of premature GCing are not great.